docs/proposals/policy.md

```diff
@@ -0,0 +1,251 @@
+
+In order to express these rules, you'll have to bind the PolicyRule to "vets" in the "cows" and "sheep" namespaces and bind it to "mechanics" in the "tractors" namespace.  For every PolicyRule you have, you'll need to create additional bindings, but you can't bind it globally or "bubba" will be allowed to inspect "sheep" and things will just get out of hand.  You really want a way to describe a set of PolicyRules that you can assign to a Team/Namespace tuple at will.  That gets you the point of containing a set of PolicyRules in something like a Role.
+
+The other problem you have is modification.  If you suddenly realize that it is too expensive to have your "vets" able to watch all of your "cows" and "sheep", you may want to change the rules to allow only "gets".  Because you instantiated PolicyRules for each Team/Namespace combination, you need to find them and update them all.  There's still a need to have the capability for an absolute rule, but it wasn't appropriate to explode it out in this case.  That gets you an object that looks something like RoleBinding where you take your set of PolicyRules and bind them a particular team/namespace while still maintain a reference to a set of rules, not a copy of them.
+
+#### Why allow TeamNames instead of simply saying that RoleBindings only have a list of UserNames?  
+Ease of maintenance and separation of powers.  Teams span namespaces, but RoleBindings do not.  If you specify the full set of users on every RoleBinding, then you have to repeat it for every namespace.  If that membership changes, you now have to update it in multiple locations.  As for the separation of powers, if you have rights to modify a RoleBinding, you have the power to change its permissions (RoleTemplateName) instead of only having the power to change its membership.  It seems reasonable to allow someone to modify team membership ("chubba is mechanic") without also having the ability to modify permissions associated with the team itself ("mechanics are super-admins").  Having Teams distinct from Roles makes it easy to express this in policy.
+
+#### Why do I have to specify a RoleBinding in each namespace it applies to?
+Because etcd doesn't have a query engine or referential integrity.  For a given request, we need to quickly locate all policy rules that apply to it.  We have structured our etcd keys to be quickly listable based on namespace.  By storing all rules that apply to a particular request under two key paths (namespaced rules or global rules), that retrieval is relatively efficient.  Storing them any other way would require an efficient index.
+
+### One off endpoints
```

Comment:
This whole section is buried in this document, yet its a fundamental addition to our API.  Can you move this higher up into a API section of this document?  You need to also capture a use case in your use cases section to capture the need for these requests more crisply.